### ISSUE TITLE
Attempting to set AppDetail on fcl.config causes decoding error. 

### ISSUE THREAD

**Issue by paulfreeman** [portto/fcl-swift#32]
This is happening in versions 0.3.1 and 0.3.0 of the SDK

In attempting to configure fcl I put an appDetail property into the config 

```
 let detail = AppDetail(title: "My Company", icon:appIconURL))
 try  fcl.config
                .put(.appDetail(detail))
```

On attempting authentication,  there is an error in AppDetail.encode, this fails at the following with an EXC error 

`var unkeyedContainer = encoder.unkeyedContainer()`

The error is occurring on the call to **encoder.unkeyedContainer()**

Detailed error is : 

Thread 22: EXC_BREAKPOINT (code=1, subcode=0x1a55c5230)

**Comment by andrew54068** [comment]
Fixed in #33. Thanks for reporting that. 🎉

**Comment by andrew54068** [comment]
I just released 0.3.2 to fix this issue.

**Comment by paulfreeman** [comment]
Thanks for the prompt action.
